src/config.rs

```diff
@@ -1903,16 +1930,36 @@ impl TiKvConfig {
             .into());
         }
 
+        rollback_or!(
```

Comment:
What is the purpose of these `rollback_or`?

Reply:
`rollback_or` are used to collect rollback of invalid config if `rb_collecter` is `Some`, or just return the `Err` provided as the last argument. The purpose of introduce `rollback_or` is to correct invalid config on PD, because user can change config on PD with no validation.

Reply:
Got it, but the name `rollback_or` here seems not appropriate, maybe validate_or_rollback is better.